hazelcast-code-generator/src/main/resources/codec-template-java.ftl

```diff
@@ -287,6 +287,9 @@ public final class ${model.className} {
             <#case "java.lang.Integer">
         ${varName} = clientMessage.getInt();
                 <#break >
+            <#case "java.lang.Long">
+        ${varName} = clientMessage.getLong();
```

Comment:
Without the case, argument type List<Long> is generating this:

`public static RequestParameters decodeRequest(ClientMessage clientMessage) {
        final RequestParameters parameters = new RequestParameters();
        java.lang.String name = null;
        name = clientMessage.getStringUtf8();
            parameters.name = name;
    
        java.util.List<java.lang.Long> hash = null;
            int hash_size = clientMessage.getInt();
            hash = new java.util.ArrayList<java.lang.Long>(hash_size);
            for (int hash_index = 0;hash_index<hash_size;hash_index++) {
                java.lang.Long hash_item;
        hash_item = clientMessage.getJava.lang.Long();
                hash.add(hash_item);
            }
            parameters.hash = hash;
    
        return parameters;
    }`

Notice: `hash_item = clientMessage.getJava.lang.Long();`